src/main/java/me/zeroeightsix/kami/module/modules/player/ViewLock.kt

```diff
@@ -0,0 +1,114 @@
+package me.zeroeightsix.kami.module.modules.player
+
+import me.zeroeightsix.kami.event.events.SafeTickEvent
+import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.setting.Setting
+import me.zeroeightsix.kami.setting.Settings
+import me.zeroeightsix.kami.util.event.listener
+import java.util.*
+import kotlin.math.*
+
+@Module.Info(
+        name = "ViewLock",
+        category = Module.Category.PLAYER,
+        description = "Locks your camera view"
+)
+object ViewLock : Module() {
+
+    private val deltaXQueue: Queue<Pair<Int, Long>> = LinkedList<Pair<Int, Long>>(listOf())
+    private val deltaYQueue: Queue<Pair<Int, Long>> = LinkedList<Pair<Int, Long>>(listOf())
```

Comment:
You don't need to specific type `Queue<Pair<Int, Long>>` here. And the redundant `listOf()` should be removed since it is doing nothing.